cypress/page/products-list.page.ts

```diff
@@ -2,20 +2,22 @@ class ProductsListPage {
   private addToCart: string;
   private containerCart: string;
   private product: string;
+  private buttonContainer: string;
 
   constructor() {
     this.containerCart = ".available-now";
-    this.addToCart = ".ajax_add_to_cart_button";
+    this.addToCart = ".ajax_add_to_cart_button span";
```

Comment:
here you can use an attribute to indicate that we are looking for the Add to cart button